app/views/layouts/application.html.erb

```diff
@@ -1,117 +1,110 @@
-            <%= form_for venues_path, method: :get do %>
-              <div class="large-8 small-9 columns">
-                <%= text_field_tag :query, params[:query], attr: "Find Stuff" %>
-              </div>
-              <div class="large-4 small-3 columns">
-                <%= submit_tag "Search", name: nil %>
-              </div>
-            <% end %>
-          </div>
-        </li>
-        <li class="has-dropdown">
-          <a href="#">Filter by Boston Neighborhood</a>
-          <ul class="dropdown">
-            <li><a href="#">Back Bay</a></li>
-            <li><a href="#">Central Square</a></li>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title><%= content_for?(:title) ? yield(:title) : "foundation-rails" %></title>
+
+    <%= stylesheet_link_tag    "application" %>
+    <%= javascript_include_tag "vendor/modernizr" %>
+    <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+    <link href="http://cdnjs.cloudflare.com/ajax/libs/foundicons/3.0.0/foundation-icons.css" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Righteous' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet' type='text/css'>
+
+    <div data-sticky-container>
+      <div class="sticky" data-sticky data-margin-top=0 style="width:100%";>
+        <nav class="top-bar" data-topbar role="navigation">
+          <ul class="title-area">
+            <li class="name">
+              <h1><a href="/" style="font-family: 'Righteous', cursive; font-size: 25px">Jump<u>e</u>r</a></h1>
+            </li>
           </ul>
-        </li>
-        <% if user_signed_in? %>
-          <% unless current_user.avatar.url.nil? %>
-            <li class="picture"><%= image_tag(current_user.avatar) %></li>
-          <% end %>
-        <li><%= link_to "Profile", "/users/#{current_user.id}" %></li>
-        <li><%= link_to "Sign Out", destroy_user_session_path, :method => :delete %></li>
-        <% else %>
-        <li><%= link_to "Sign In", new_user_session_path %></li>
+          <section class="top-bar-section">
+            <!-- Right Nav Section -->
+            <ul class="right">
+              <li class="has-form">
+                <div class="row collapse" style="background-color: #ee6f19">
+                  <%= form_tag("/search", method: "get") do %>
+                    <div class="large-8 small-9 columns">
+                      <%= text_field_tag :query, params[:query], attr: "Find Stuff" %>
+                    </div>
+                    <div class="large-4 small-3 columns">
+                      <%= submit_tag "Search", name: nil %>
+                    </div>
+                  <% end %>
+                </div>
+              </li>
```

Comment:
Extract lines 28-39 into a partial `views/application/_top_bar_form.html.erb` You can render this partial with `<%= render "top_bar_form" %>`